app/Http/Requests/CelebRequest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class CelebRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return auth()->check() && auth()->user()->is_admin;
```

Comment:
This one should check if the is_admin is === true :D otherwise it would not matter if it is false or not